pom.xml

```diff
@@ -1505,6 +1525,7 @@
                             <!--@TODO After fixing https://github.com/apache/druid/issues/4964 remove this parameter-->
                             -Ddruid.indexing.doubleStorage=double
                         </argLine>
+                        <skipTests>${skipUTs}</skipTests>
```

Comment:
Is this okay? Wouldn't this end up skipping all tests and not just UTs?

Reply:
Added a comment. Surefire runs the UTs. It sister plugin, Failsafe, runs the ITs. Here, we want to skip the Surefire tests only. Let me know if the comment makes this clear, else I'll add to it.